e2e/testdata/fn-render/invalid-configmap-fnconfig/.expected/config.yaml

```diff
@@ -13,4 +13,6 @@
 # limitations under the License.
 
 exitCode: 1
-stdErr: "line 14: cannot unmarshal !!seq into string"
\ No newline at end of file
+stdErr: |
+  Error: unable to parse "Kptfile": yaml: unmarshal errors:
+    line 14: cannot unmarshal !!seq into string
```

Comment:
@mortent submitted another PR that improves the handling of kptfile parsing/missing related errors, so reconcile with that. https://github.com/GoogleContainerTools/kpt/pull/1977